util/reggen/ip_block.py

```diff
@@ -331,6 +346,9 @@ def _asdict(self) -> Dict[str, object]:
 
         ret['scan_reset'] = self.scan_reset
 
+        if self.attr:
```

Comment:
It doesn't matter too much because we don't use empty strings for "attr" parameters, but I think this would be clearer with `if self.attr is not None`.